DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -1463,7 +1466,6 @@ function RunTests()
 
     testint(BingoActiveMission(1, 0), 1, "BingoActiveMission maybe");
     testint(BingoActiveMission(1, (1<<1)), 2, "BingoActiveMission");
-    testint(BingoActiveMission(2, (1<<1)), -1, "BingoActiveMission too late");
```

Comment:
we should bring this test back, the reason you had to make the 2nd commit was because you were too quick to delete this test in the first place so obviously it has value